ampersand-form-view.js

```diff
@@ -31,6 +31,12 @@ module.exports = View.extend({
         }
     },
 
+    template: [
+        '<form>',
+            '<fieldset data-hook="field-container"></fieldset>',
```

Comment:
Good point, particularly the last one. In that case, I can update it to remove the `<fieldset>` tag and put `data-hook="field-container"` in the `<form>` tag.